### ISSUE TITLE
Tensorboard in detection

### ISSUE THREAD

**Issue by nowtransform** [open-mmlab/mmdetection#10521]
How can I get tensorboard logs during training?

**Comment by hujh1994** [comment]
https://github.com/open-mmlab/mmengine/blob/04b0ffee76c41d10c5fd1f737cdc5306af365754/docs/en/common_usage/visualize_training_log.md#tensorboard

**Comment by SeanBackstrom** [comment]
As per the documentation above, in your config change the 
```vis_backends=[dict(type='TensorboardVisBackend')]```

then in a notebook you can run
```
%load_ext tensorboard
%tensorboard --logdir mmdetection/work_dirs
```
then whatever port its using you can go to your browser and go to localhost:PORT

**Comment by XinyueZ** [comment]
@hujh1994 I've discerned that the default displays the model's performance on the training set. How can I configure it to reflect the validation set, or even the test set?
I think the project is exceptional, but the documentation is somewhat lackluster at best.

thank you
